scripts/ci/publish-artifacts

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -eo pipefail
+
+if [[ -n $DRONE_TAG ]]; then
+	REF=$DRONE_TAG
+	TAG=1
+elif [[ -n $DRONE_BRANCH ]]; then
+	REF=$DRONE_BRANCH
+	TAG=0
+else
+	# Like, really. It'll overwrite your SSH key.
+	echo "This script doesn't work outside of Drone" >&2
+	exit 1
+fi
+
+# Upload packages to repo.pyr8.io
+
+(
+umask 077
+mkdir -p ~/.ssh
+echo "$SSH_PRIVATE_KEY" | base64 -d | tr -d '\r' >~/.ssh/id_rsa
+)
```

Comment:
Why is this executed in a subshell?

Reply:
`chmod` would be more explicit here then

Reply:
I'd have to do it twice.

Reply:
Small price to pay for improved readability.  Code is read multiple times by multiple people, most often written once by one person.

Reply:
While I prefer being explicit in code I understand both sides of the issue. It is just as easy to look up what is going on. You can always add a comment to the side to aid understnading. i.e. <code> # some comment. In general, if it functionality of the code works and there is no definite issue with code I'll commit and then you can evolve the code in next PR. These issues seem to be hard to resolve mutually on small scripts. As it grows it becomes evident what needs to be done.